.circleci/real_config.yml

```diff
@@ -2141,6 +2141,9 @@ jobs:
       - setup-python-venv:
           executor: <<pipeline.parameters.machine-image>>
       - setup-go-intg-deps
+      - kubernetes/install-kubectl
+      - run: curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && sudo install minikube-linux-amd64 /usr/local/bin/minikube
```

Comment:
It'd be nice to use the minikube orb at https://circleci.com/developer/orbs/orb/ccpgames/minikube for the installation.  Behind the scenes it does basically the same thing, but by moving the installation process out into an orb that someone else maintains, there's a bit less for us to worry about longer-term.  And the config becomes easier to read, since it's just "minkube/minikube-install" instead of some big pile of `curl && install` :)

Reply:
ultimately, not going the orb route :(